nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/test/comment.test.js

```diff
@@ -160,19 +164,19 @@ describe("Excluding all variables", () => {
     });
 });
 
-describe("With tracing", () => {
+describe("With OpenCensus tracing", () => {
```

Comment:
As mentioned above, we should also test for the default case. ..and if keeping the two levels of default handling, test against an unknown provider string as well.

Reply:
Ack, please take a look at @kintel's suggestion here.

Reply:
Added